src/test/resources/logback.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<configuration>
+
+	<appender name="LE"
+		class="com.rapid7.logback.LogentriesAppender">
+		<Token>0c7407d4-fd0d-4436-bb50-44f1266b4490</Token>
+		<Region>eu</Region>
+		<Debug>False</Debug>
+		<Ssl>true</Ssl>
+		<facility>USER</facility>
+		<dataHubAddr>localhost</dataHubAddr>
+		<dataHubPort>20000</dataHubPort>
+		<debug>true</debug>
```

Comment:
`Debug = False` and `debug = true`? what does it mean?

Reply:
that's wrong, I'll fix it.